src/bin/pants/main.rs

```diff
@@ -233,6 +237,26 @@ fn get_summary_message(component_count: u32, vulnerability_count: u32) -> String
         "\nAudited Dependencies: {}\nVulnerable Dependencies: {}\n",
         component_count, vulnerability_count
     );
+
+    // Creating the overall table for displaying the summary of the scan
+    let mut table = Table::new();
+    table.max_column_width = 40;
+    if cfg!(windows) {
+        table.style = term_table::TableStyle::simple();
+    } else {
+        table.style = term_table::TableStyle::rounded();
+    }
+    table.add_row(Row::new(vec![TableCell::new("Summary")]));
+    table.add_row(Row::new(vec![
+        TableCell::new("Audited Dependencies"),
+        TableCell::new_with_col_span(component_count as u32, 1),
```

Comment:
minor: could remove explicit cast: as u32
```suggestion
        TableCell::new_with_col_span(component_count, 1),
```

Reply:
Yeah, you are right. The as u32 is obsolete there. I am happy that you have seen it.